langchain_mcp_adapters/tools.py

```diff
@@ -150,6 +150,17 @@ def convert_mcp_tool_to_langchain_tool(
         msg = "Either a session or a connection config must be provided"
         raise ValueError(msg)
 
+    input_schema = None
```

Comment:
@FerroLx thanks for the PR: The place to fix this is likely langchain_core. Could you check if the issue exists in langchain_core / if you can reproduce it there?